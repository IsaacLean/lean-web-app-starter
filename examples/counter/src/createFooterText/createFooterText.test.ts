import createFooterText from ".";

test("Footer text element matches snapshot", () => {
  expect(createFooterText()).toMatchSnapshot();
});
