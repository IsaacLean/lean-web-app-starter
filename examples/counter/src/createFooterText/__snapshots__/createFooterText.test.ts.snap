// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer text element matches snapshot 1`] = `
<footer
  class="text-center text-muted"
  id="footer-text"
>
  <p
    class="mb-0"
  >
    Learn about the tech stack used for this project and more at its 
    <a
      class="text-muted"
      href="https://github.com/mattlean/lean-js-app-starter/tree/v1.0.0/examples/counter"
    >
      GitHub repo
    </a>
    .
  </p>
  <p>
    Built with 
    <a
      class="text-muted"
      href="https://github.com/mattlean/lean-js-app-starter"
    >
      Lean JS App Starter
    </a>
    .
  </p>
</footer>
`;
